core/core/src/ansible/roles/deployments/templates/auth-service/auth-service.yml.j2

```diff
@@ -0,0 +1,242 @@
+---
+
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ namespace_name }}
+
+---
+
+apiVersion: v1
+data:
+  password: "{{ data.database.password | b64encode }}"
+kind: Secret
+metadata:
+  labels:
+    app: {{ auth_service_name }}
+  name: {{ auth_service_name }}-db
+  namespace: {{ namespace_name }}
+type: Opaque
+
+---
+
+apiVersion: v1
+data:
+  password: "{{ data.service.admin_password | b64encode }}"
+kind: Secret
+metadata:
+  labels:
+    app: {{ auth_service_name }}
+  name: {{ auth_service_name }}-http
+  namespace: {{ namespace_name }}
+type: Opaque
+
+---
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  labels:
+    app: {{ auth_service_name }}
+  name: {{ auth_service_name }}
+  namespace: {{ namespace_name }}
+data:
+  keycloak.cli: |
+    embed-server --server-config=standalone-ha.xml --std-out=echo
+    batch
+
+    # Makes node identifier unique getting rid of a warning in the logs
+    /subsystem=transactions:write-attribute(name=node-identifier, value=${jboss.node.name})
+
+    # Allow log level to be configured via environment variable
+    /subsystem=logging/console-handler=CONSOLE:write-attribute(name=level, value=${env.WILDFLY_LOGLEVEL:INFO})
+    /subsystem=logging/root-logger=ROOT:write-attribute(name=level, value=${env.WILDFLY_LOGLEVEL:INFO})
+
+    # Log only to console
+    /subsystem=logging/root-logger=ROOT:write-attribute(name=handlers, value=[CONSOLE])
+
+    /socket-binding-group=standard-sockets/socket-binding=proxy-https:add(port=443)
+    /subsystem=undertow/server=default-server/http-listener=default:write-attribute(name=redirect-socket, value=proxy-https)
+    /subsystem=undertow/server=default-server/http-listener=default:write-attribute(name=proxy-address-forwarding, value=true)
+
+    /subsystem=infinispan/cache-container=keycloak/distributed-cache=sessions:write-attribute(name=owners, value=${env.CACHE_OWNERS:2})
+    /subsystem=infinispan/cache-container=keycloak/distributed-cache=authenticationSessions:write-attribute(name=owners, value=${env.CACHE_OWNERS:2})
+    /subsystem=infinispan/cache-container=keycloak/distributed-cache=offlineSessions:write-attribute(name=owners, value=${env.CACHE_OWNERS:2})
+    /subsystem=infinispan/cache-container=keycloak/distributed-cache=loginFailures:write-attribute(name=owners, value=${env.CACHE_OWNERS:2})
+
+    /subsystem=jgroups/channel=ee:write-attribute(name=stack, value=tcp)
+
+    run-batch
+    stop-embedded-server
+  keycloak.sh: |
+    #!/usr/bin/env bash
+
+    set -eu
+
+    /opt/jboss/keycloak/bin/jboss-cli.sh --file=/scripts/keycloak.cli
+
+    exec /opt/jboss/tools/docker-entrypoint.sh -b 0.0.0.0  -c standalone-ha.xml
+    exit "$?"
```

Comment:
$? is exit code (always integer) so I think quotation marks are not needed

Reply:
But also are not bad here.